tests/SideBySide/LoadDataInfileAsync.cs

```diff
@@ -42,17 +42,35 @@ public LoadDataInfileAsync(DatabaseFixture database)
 			Assert.Equal(20, rowCount);
 		}
 
-		[BulkLoaderLocalCsvFileFact]
+		[BulkLoaderLocalCsvFileFact(TrustedHost = true)]
 		public async void CommandLoadLocalCsvFile()
 		{
-			string insertInlineCommand = string.Format(m_loadDataInfileCommand, " LOCAL", AppConfig.MySqlBulkLoaderLocalCsvFile.Replace("\\", "\\\\"));
+			string insertInlineCommand = string.Format(m_loadDataInfileCommand, " LOCAL",
+				AppConfig.MySqlBulkLoaderLocalCsvFile.Replace("\\", "\\\\"));
 			MySqlCommand command = new MySqlCommand(insertInlineCommand, m_database.Connection);
-			if (m_database.Connection.State != ConnectionState.Open) await m_database.Connection.OpenAsync();
+
+			if (m_database.Connection.State != ConnectionState.Open)
+				await m_database.Connection.OpenAsync();
+
 			int rowCount = await command.ExecuteNonQueryAsync();
+
 			m_database.Connection.Close();
 			Assert.Equal(20, rowCount);
 		}
 
+		[BulkLoaderLocalCsvFileFact(TrustedHost = false)]
```

Comment:
You can put `#if !BASELINE` around this whole test, since this is a deliberate breaking change with the "baseline" connector (i.e., Oracle's Connector/NET) so we're not worried about testing backwards compatibility.

Reply:
I have missed that. Thank you!